docs/source/examples/supplychain/supplychain_transaction_family.rst

```diff
@@ -43,61 +43,121 @@ Agent
 Agents represent owners or custodian of goods in the system. All agents must
 be registered in the system before they can interact with records.
 
-.. literalinclude:: ../../../../families/supplychain/protos/agent.proto
```

Comment:
It would seem more obvious when reviewing the documents generated output to spot an incorrect offset in an include than a line in the explicit protobuf definition that is of sync with the protobuf definition.